appsdk/sdk.go

```diff
@@ -105,13 +114,68 @@ func (sdk *AppFunctionsSDK) MakeItRun() error {
 	return nil
 }
 
+// LoadConfigurablePipeline ...
+func (sdk *AppFunctionsSDK) LoadConfigurablePipeline() ([]AppFunction, error) {
+
+	sdk.usingConfigurablePipeline = true
+
+	configurable := AppFunctionsSDKConfigurable{
+		Sdk: sdk,
+	}
+
+	var pipeline []AppFunction
+	valueOfType := reflect.ValueOf(configurable)
+	pipelineConfig := configurable.Sdk.Pipeline()
+
+	for _, functionName := range strings.Fields(pipelineConfig.ExecutionOrder) {
```

Comment:
For consistency, should the  ExecutionOrder list be separated by non-white-space?